detox/src/utils/easy-file-io.test.js

```diff
@@ -0,0 +1,54 @@
+describe('Easy-file I/O utilities', () => {
+  describe('For base-64 encoded data', () => {
+    const data = 'VGhlcmUgaXMgbm8gc3Bvb24h'; // There is no spoon!
+    const filePath = 'path/to/file.ext'
+    const tempFilePath = '/path/to/temp-file.ext';
+
+    const writtenBuffer = () => fs.writeFileSync.mock.calls[0][1];
+
+    let fs;
+    let tempfile;
+    let uut;
+    beforeEach(() => {
+      jest.mock('fs');
+      fs = require('fs');
+
+      jest.mock('tempfile');
+      tempfile = require('tempfile');
+      tempfile.mockReturnValue(tempFilePath);
+
+      uut = require('./easy-file-io');
+    });
+
+    it('should save raw data to a given file', () => {
```

Comment:
Where do you use this scenario, Amit? @d4vidi

Reply:
Right. I'll remove support for that.